webapp/src/views/userSettings/userProfile/UserProfileView.tsx

```diff
@@ -58,37 +63,53 @@ export const UserProfileView: FunctionComponent = () => {
 
     return (
       <Box data-cy="user-profile">
-        <UserProfileAvatar />
-        <TextField
-          variant="standard"
-          name="name"
-          label={<T>User settings - Full name</T>}
-        />
-        <TextField
-          variant="standard"
-          name="email"
-          label={<T>User settings - E-mail</T>}
-        />
-        {user?.emailAwaitingVerification && (
-          <Box>
-            <Typography variant="body1">
-              <T
-                parameters={{
-                  email: user.emailAwaitingVerification!,
-                }}
-              >
-                email_waiting_for_verification
-              </T>
-            </Typography>
-          </Box>
-        )}
+        <Grid container spacing={8}>
+          <Grid item xs="auto">
+            <UserProfileAvatar />
+          </Grid>
+          <Grid item xs={12} sm>
+            <TextField
+              variant="standard"
+              name="name"
+              label={<T>User settings - Full name</T>}
+            />
+            <TextField
+              variant="standard"
+              name="email"
+              disabled={isManaged}
+              helperText={isManaged ? t('managed-account-field-hint') : void 0}
```

Comment:
Why not `: undefined`? This is a bit unusual.

Reply:
`void 0` is something I'm used to write as it's shorter, and because of how JavaScript works; tldr it's to avoid shadowing risk and because it skips the lookup step in the V8 engine. That being said, I can write it as `undefined` if it better suits the code style